apps/generator/docs/usage.md

```diff
@@ -4,34 +4,19 @@ weight: 30
 ---
 
 There are two ways to use the generator:
-- [AsyncAPI CLI](#generator-cli)
-- [Generator library](#using-as-a-modulepackage)
+- [AsyncAPI CLI](#asyncapi-cli)
+  - [`asyncapi generate fromTemplate ASYNCAPI TEMPLATE`](#asyncapi-generate-fromtemplate-asyncapi-template)
+  - [Global templates installed with `yarn` or `npm`](#global-templates-installed-with-yarn-or-npm)
+  - [CLI usage examples](#cli-usage-examples)
+  - [CLI usage with Docker](#cli-usage-with-docker)
+  - [CLI usage with `npx` instead of `npm`](#cli-usage-with-npx-instead-of-npm)
```

Comment:
do we need this?

navigation is automatically rendered anyway - look https://www.asyncapi.com/docs/tools/generator/usage